src/reducers/rdf.js

```diff
@@ -0,0 +1,72 @@
+const N3 = require('n3')
+
+const { DataFactory } = N3
+// TODO: Use N3.Store() for graph state
+const { namedNode, literal, blankNode, quad } = DataFactory
+
+const DEFAULT_STATE = {
+  graph: {}
+}
+
+const addTriples = (subject, formData, rtId, testProperty, graph) => {
```

Comment:
I wouldn't mind seeing if we can refactor this so that we are not always passing in all these arguments. Maybe separate functions to handle 'uri' and 'content'/other (e.g. addTriplesForUri())? I'd be happy to work on this as part of a follow-on PR. Non-blocker for this PR.

Reply:
Yeah, not sure what the best option is here; originally I had this functionality within the `generateMyRDF` with two forEach loops.